harness/determined/tensorboard/fetchers/shared.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import logging
+import os
+import posixpath
+import shutil
+from typing import Any, Dict, Generator, List, Tuple
+
+from .base import Fetcher
+
+logger = logging.getLogger(__name__)
+
+
+class SharedFSFetcher(Fetcher):
```

Comment:
Just had a thought: Why even have a SharedFS Fetcher?  Why not just use the files right in the shared_fs?

Reply:
Good-ish reasons:
Consistency.
I can't think of a good reason to directly access `shared_fs`.
I do think that there will be an odd exception in the tensorboard code for shared_fs in which it needs to worry about the storage type.

Not-so-good-reasons:
I would like this to go in by Wednesday. Can we make that change at a later point?

If required for your approval, please tell me and I will make the change.

Reply:
> I can't think of a good reason to directly access shared_fs.
Less code, and less disk io?

> Can we make that change at a later point?
Yes, I think that's fine.